### PR TITLE
Update detray to v0.81.0

### PR DESCRIPTION
**Pull request by stephenswat**
Brings changes to the stepper states and the wire chambers.

### CODE REVIEW

File: core/include/traccc/finding/actors/ckf_aborter.hpp [hunk 1]
```diff
@@ -41,7 +41,7 @@ struct ckf_aborter : detray::actor {
 
         // Abort at the next sensitive surface
         if (navigation.is_on_sensitive() &&
-            stepping._s > abrt_state.min_step_length) {
+            stepping.step_size() > abrt_state.min_step_length) {
```
Comment (niermann999) [inline core/include/traccc/finding/actors/ckf_aborter.hpp:44]:
```suggestion
            stepping.path_from_surface() > abrt_state.min_step_length) {
```

**Review by niermann999 — CHANGES_REQUESTED** [review]

File: core/include/traccc/finding/actors/ckf_aborter.hpp (hunk 1 above)
Comment (stephenswat) [reply core/include/traccc/finding/actors/ckf_aborter.hpp]: Fixed, thanks!

**Review by niermann999 — APPROVED** [review]

**Review by krasznaa — APPROVED** [review]
Sure, all looks reasonable. 🤔

**Comment by stephenswat** [comment]
I'm getting a bit tired of the OneAPI compiler running into some internal error every other week... :frowning_face:

**Comment by stephenswat** [comment]
Turns out that the current version of detray, 0.81.0, manages to resolve the compiler bug.

**Review by niermann999 — APPROVED** [review]

**Pull request merged**